src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -43,13 +43,13 @@
         <Value>$(BridgeRemoteEnabled)</Value>
         <DefaultValue>false</DefaultValue>
       </TestProperty>
-      <TestProperty Include="BridgeCertificateAuthority">
-        <Value>$(BridgeCertificateAuthority)</Value>
-        <DefaultValue>WcfBridge_RootCA.cer</DefaultValue>
+      <TestProperty Include="BridgeCertificatePassword">
+        <Value>$(BridgeCertificatePassword)</Value>
+        <DefaultValue>test</DefaultValue>
       </TestProperty>
-      <TestProperty Include="BridgeHttpsCertificate">
-        <Value>$(BridgeHttpsCertificate)</Value>
-        <DefaultValue>WcfBridge_localhost.pfx</DefaultValue>
+      <TestProperty Include="BridgeCertificateValidityPeriod">
+        <Value>$(BridgeCertificateValidityPeriod)</Value>
+        <DefaultValue>1.00:00:00</DefaultValue>
```

Comment:
It's already done :smile: The reason why it's weird is because the usage pattern is weird. The earlier field you commented on in CertificateGenerator was a default value. Once the Bridge configuration is pushed over, Initialize() is called - and that's what determines the real initialized values in the CertificateGenerator.